tooling/src/hypothesistooling/__main__.py

```diff
@@ -435,9 +435,9 @@ def run_tox(task, version, *args):
     "3.9": "3.9.19",
     "3.10": "3.10.14",
     "3.11": "3.11.9",
-    "3.12": "3.12.3",
+    "3.12": "3.12.4",
     "3.13": "3.13.0b2",
-    "3.14": "3.14-dev",
+    "3.14": "3.14t-dev",
     "pypy3.8": "pypy3.8-7.3.11",
```

Comment:
what does the `t` denote in `3.14t-dev` 🤔 ? 

```python
from packaging.version import Version

# packaging.version.InvalidVersion: Invalid version: '3.14t-dev'
Version("3.14t-dev")
```

Reply:
It's [the free-threading build](https://github.com/pyenv/pyenv/blob/master/plugins/python-build/share/python-build/3.14t-dev)!  We're [not actually running 3.14 in CI yet](https://github.com/HypothesisWorks/hypothesis/blob/d8c17832141f587fec9ef89895ed01a4e9c1650d/.github/workflows/main.yml#L52-L54), but we should probably / eventually:

- [ ] fix our auto-update script so it defaults to the standard build
- [ ] start running CI on 3.14 standard
- [ ] add another set of builds for the free-threading version and run that too